### ISSUE TITLE
Docs: Feedback for Get Started with SQL Over Maps

### ISSUE THREAD

**Issue by naadhira** [hazelcast/hz-docs#283]
Hi, I have some feedback about [this page](https://github.com/hazelcast/hz-docs/edit/v/5.0/docs/modules/sql/pages/get-started-sql.adoc)

Step 2.11 is not a good example of the concept of using HAVING for aggregated data. 

The stated objective is to get a list of cities with a population of over 1 million,. You can do that with a simple WHERE query.
```
select population2020.__key as large_cities
   > from population2020
   > where population2020.this >= 1000000;
```

There's no aggregated data here. I don't think there's an easy way to do aggregation with the data provided. My suggestion is to add more cities in the same country, join the tables, do a SUM using GROUP BY country, then look for *countries* with population counts over (some number).

**Comment by JakeSCahill** [comment]
Could you provide the SQL query for what you mean?

**Comment by naadhira** [comment]
It's Step 2.12 in the tutorial: 
```
SELECT population2020.__key AS large_cities
FROM population2020
GROUP BY population2020.__key HAVING SUM(population2020.this) > 1000000;

```
The problem here is that there's nothing to group. The key in this case is the name of a city, and each key/city is unique, so there's no actual aggregation happening. If we want aggregation, we have to change the data we're using so that there's some common value to group on. 

Short term fix is to drop step 2.12 altogether until the tutorial can be rewritten with better data that provides something to aggregate.

**Comment by JakeSCahill** [comment]
Will be fixed by https://github.com/hazelcast/hz-docs/pull/294
